src/documentation/reference/ReferenceNode.tsx

```diff
@@ -91,27 +99,40 @@ const ReferenceNode = ({
     disclosure,
     fullName,
   ]);
-
+  const handleHighlightClick = useCallback(() => {
+    setHighlighting(false);
+  }, [setHighlighting]);
   return (
     <Box
       ref={ref}
       id={fullName}
       wordBreak="break-word"
       mb={kindToSpacing[kind]}
-      {...others}
       _hover={{
         "& button": {
           display: "flex",
         },
       }}
       fontSize="sm"
+      {...others}
     >
-      <ReferenceNodeSelf
-        docs={docs}
-        showMore={disclosure.isOpen}
-        onToggleShowMore={disclosure.onToggle}
-      />
-      <ReferenceNodeChildren docs={docs} anchor={anchor} />
+      <Highlight
+        onClick={handleHighlightClick}
+        value={highlighting}
+        mt={1}
+        mb={1}
+        pl={3}
+        pr={3}
+        pt={3}
+        pb={3}
```

Comment:
Just `p={3}`?

Reply:
Hah, yes, they started different and I hadn't noticed they'd ended up the same. Thanks.

Reply:
85f378547cef568cbf8d3e3d5401cbab3bdb341f